lnrpc/rpc.proto

```diff
@@ -1845,12 +1845,19 @@ message FundingPsbtFinalize {
     /*
     The funded PSBT that contains all witness data to send the exact channel
     capacity amount to the PK script returned in the open channel message in a
-    previous step.
+    previous step. Cannot be set at the same time as final_raw_tx.
     */
     bytes signed_psbt = 1;
 
     // The pending channel ID of the channel to get the PSBT for.
     bytes pending_chan_id = 2;
+
+    /*
+    As an alternative to the signed PSBT with all witness data, the final raw
+    wire format transaction can also be specified directly. Cannot be set at the
+    same time as signed_psbt.
+    */
+    bytes final_raw_tx = 3;
```

Comment:
I guess using a oneof here would be a breaking change, which we don't want.

Reply:
I assumed so, yes. But I can investigate if perhaps it isn't since the numerical IDs would remain the same if we put the two parameters in a `oneof`.

Reply:
I think there still would be due to the name of the field changing as part of a oneof.

Reply:
Right, it would definitely break at compile-time. But perhaps it would still be compatible at the wire level. But probably not worth investigating at this point then.